orchagent/orch.cpp

```diff
@@ -866,6 +878,7 @@ void Orch2::doTask(Consumer &consumer)
     while (it != consumer.m_toSync.end())
     {
         bool erase_from_queue = true;
+        ReturnCode rc;
```

Comment:
We need to revert this as well. Should be no change.

Reply:
Fixed